api/TextDecoder.json

```diff
@@ -5,63 +5,75 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextDecoder",
         "support": {
           "webview_android": {
```

Comment:
Ideally `webview_android` would be moved to the bottom so it's alphabetized but that's not a deal breaker imo